packages/web3-eth-ens/src/config.ts

```diff
@@ -1 +1,22 @@
 export const registryContractAddress = '0x00000000000C2E074eC69A0dFb2997BA6C7d2e1e';
+
+// https://docs.ens.domains/contract-developer-guide/writing-a-resolver
+// resolver interface Ids
+export const interfaceIds: { [T: string]: string } = {
+	addr: '0x3b3b57de',
+	name: '0x691f3431',
+	abi: '0x2203ab56',
+	pubkey: '0xc8690233',
+	text: '0x59d1d43c',
+	contenthash: '0xbc1c58d1',
```

Comment:
in the docs it suggests to support `supportsInterface` with the interface id of `0x01ffc9a7`